stack.yaml

```diff
@@ -81,7 +81,7 @@ packages:
 # https://github.com/well-typed/cborg/issues/171
 - location:
     git: https://github.com/well-typed/cborg
-    commit: 3d274c14ca3077c3a081ba7ad57c5182da65c8c1
+    commit: 37de2c0ae6bc7fa53f24f8d13748a7c370da9149
```

Comment:
Can you update the comment on this one indicating the PR that we're waiting on getting merged upstream?

Reply:
Looks like this has been merged/released already. Let's try.